vyxal/elements.py

```diff
@@ -3013,6 +3013,46 @@ def zero_slice(lhs, rhs, ctx):
     }.get(ts, lambda: vectorise(zero_slice, lhs, rhs, ctx=ctx))()
 
 
+def nth_pi(lhs, ctx):
+    """Element ∆i
+    (int) -> nth_pi(a)
+    """
+    return pi_digits(lhs)[lhs]
+
+
+def nth_e(lhs, ctx):
+    """Element ∆ė
+    (int) -> nth_e(a)
+    """
```

Comment:
This should be made to vectorise over lists, which can be done by checking if `vy_type(lhs, simple=True) is list`